### DISCUSSION TITLE
Dropdown input options can load from dim table, no need input it manually

### DISCUSSION THREAD

**Discussion by mggger** [directus/directus#15978] [Archive]
![4C062401-8BD5-403A-BA01-C9AEFCE2B97F](https://user-images.githubusercontent.com/13358284/195520760-2c2faa26-a3a2-488b-b808-dd80d292b7e3.png)

**Comment by br41nslug** [comment]
To fill a dropdown based on a table you can use a m2o relation interface instead :)

**Reply by northern-mindset** [reply]
Can't find this option for my O2M field. Only one Interface option is there, which works as a fly-out panel in the content editor, not a dropdown. Is it possible to have a small dropdown instead of a panel?

![image](https://github.com/directus/directus/assets/6065071/69c321fd-7b81-4cf7-8b0c-68ff5aefabc5)

Directus v10.2.1
